src/environment/environment.dev.ts

```diff
@@ -3,6 +3,7 @@ import { EnvironmentFile } from './models/environment.model';
 export const environment: EnvironmentFile = {
   isRemote: true,
   configUrl: 'https://dev.mes.dev-dvsacloud.uk/v1/configuration/dev',
+  approvedDeviceIdentifiers: ['iPad7,4'],
```

Comment:
Isn't this config that is going to live in the configuration service? If so, shouldn't it be in the `AppConfig` type, but not the environment model? However, I see that you check the device type before getting configuration from the backend. Have we considered moving the config call to before the device type check?

Reply:
It should be in the LocalEnviromentModel and appear in local.env as we don't call out to the config service there. It doesn't need to be in any of the others.

Reply:
it looks like I didn't fully understand how the config operated.  I do now :) so have modified accordingly.